examples/e03_struct_utilities/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "04_message_builder"
```

Comment:
Ignore the `Cargo.toml` diffs for these two examples. I do not know why git swapped the files and considers them "renamed".